tests/features/page-header.feature

```diff
@@ -63,3 +63,17 @@ Feature: Page header block component.
     Then I should see "My summary" in the "page header intro"
     And I should see "News" in the "page header meta"
     And I should see "02 April 2019" in the "page header meta"
+
+  Scenario: The page header block shows the content language switcher.
+    Given the following "Spanish" translation for the "Robots are everywhere" demo page:
+      | Title | Los robots estan en todas partes  |
+    And  I am an anonymous user
+    When I visit the "Spanish" translation page for the "Robots are everywhere" demo page
+    Then I should see the heading "Los robots estan en todas partes" in the "page header"
+    And I should not see the link "español" in the "page header" region
+
+    When I visit the "French" translation page for the "Robots are everywhere" demo page
+    Then I should see the heading "Robots are everywhere" in the "page header"
+    And I should see "français" in the "language page switcher unavailable" region
+    And I should see "English" in the "language page switcher selected" region
+    And I should see the link "español" in the "language page switcher" region
```

Comment:
The ` region` part is actually optional, we can remove it and improve the scenario readability, as shown below:
```gherkin
    And I should see "français" in the "unavailable languages in the language page switcher"
    And I should see "English" in the "available languages in the language page switcher"
```
This means we will also need to rename the two aliases above in the transformation context configuration.